feature-libs/quote/components/actions/confirm-dialog/quote-actions-confirm-dialog.component.html

```diff
@@ -83,5 +83,16 @@
         </button>
       </div>
     </div>
+
+    <!-- For screen reader purposes (not visual)-->
+    <div
+      class="cx-visually-hidden"
+      aria-atomic="true"
+      aria-live="assertive"
+      role="alert"
```

Comment:
I would remove role=alert and set aria-live to "polite" like in the add-to-cart dialog (see added-to-cart-dialog.component.html). So that both modals are similar in terms of accessibility.